feature-libs/asm/components/asm-create-customer-form/asm-create-customer-form.component.html

```diff
@@ -0,0 +1,113 @@
+        <div class="cx-dialog-header modal-header">
+          <h2 class="title modal-title">
+            {{ 'asm.createCustomerForm.title' | cxTranslate }}
+          </h2>
+          <ng-template *ngTemplateOutlet="closeButton"></ng-template>
+        </div>
+        <!-- Modal Body -->
+        <div class="modal-body">
+          <div class="row justify-content-center">
+            <div class="col-md-12">
+              <div class="form-group">
+                <label>
+                  <span class="label-content">{{
+                    'asm.createCustomerForm.firstName.label' | cxTranslate
+                  }}</span>
+                  <input
+                    required="true"
+                    class="form-control"
+                    type="text"
+                    name="firstname"
+                    placeholder="{{
+                      'asm.createCustomerForm.firstName.placeholder' | cxTranslate
+                    }}"
+                    formControlName="firstName"
+                  />
+                  <cx-form-errors
+                    [control]="registerForm.get('firstName')"
+                  ></cx-form-errors>
+                </label>
+              </div>
+
+              <div class="form-group">
+                <label>
+                  <span class="label-content">{{
+                    'asm.createCustomerForm.lastName.label' | cxTranslate
+                  }}</span>
+                  <input
+                    required="true"
+                    class="form-control"
+                    type="text"
+                    name="lastname"
+                    placeholder="{{
+                      'asm.createCustomerForm.lastName.placeholder' | cxTranslate
+                    }}"
+                    formControlName="lastName"
+                  />
+                  <cx-form-errors
+                    [control]="registerForm.get('lastName')"
+                  ></cx-form-errors>
+                </label>
+              </div>
+
+              <div class="form-group">
+                <label>
+                  <span class="label-content">{{
+                    'asm.createCustomerForm.emailAddress.label' | cxTranslate
+                  }}</span>
+                  <input
+                    required="true"
+                    class="form-control"
+                    type="email"
+                    name="email"
+                    placeholder="{{
+                      'asm.createCustomerForm.emailAddress.placeholder' | cxTranslate
+                    }}"
+                    formControlName="email"
+                  />
+                  <cx-form-errors
+                    [control]="registerForm.get('email')"
+                  ></cx-form-errors>
+                </label>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- Modal Footer -->
+        <div class="modal-footer">
+          <button type="submit" class="btn btn-primary">
```

Comment:
I'm a little confused, Do you want to say that this is **not** a good UX interaction?  if answer is YES, Do you mean that when the form has been touched and is invalid，the button should be disabled?

Reply:
Sorry, I see how that was ambiguous.  

I am suggesting that a possible better UX interaction would be: Disable the button while the form is invalid and has been touched.  

I recommend discussing this suggestion with the UX designer to see if they agree.  I only suggested this because I have had past experience where this same form interaction was requested after implementation.

Reply:
This is a topic discussed internally, and the answer is no, here is a good explanation in [the es-spa-asmcreateaccount channel](https://sap-cx.slack.com/archives/C04GTG77PJN/p1672375233430669). 
![image](https://user-images.githubusercontent.com/126219214/221123808-562576f4-b198-430f-9d74-fd0c64031da5.png)

Reply:
So it was!  Now I know my experience was out of the ordinary.  Thank you!